recipes/wisdem/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set version = "2.0.1" %}
+{% set name = "wisdem" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/WISDEM/WISDEM/archive/v{{ version }}-alpha.tar.gz
+  sha256: dae8b7d846282d8f0b7d856e79771cf03d5cdfce726c0b765453f36b9fa2aca8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - pip
+    - python >=3.7
+    - numpy
+    - setuptools
+    - cython
+    - scipy
+  run:
+    - python >=3.7
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - matplotlib
```

depend on `matplotlib-base`. `matplotlib` is an end user convenience package of `matplotlib-base`and the Qt backend.
```suggestion
    - matplotlib-base
```